app/assets/scss/_pagination.scss

```diff
@@ -0,0 +1,79 @@
+nav[role=navigation] {
+display: block;
+}
+
+.previous-next-navigation {
```

Comment:
It looks like this copied straight from the buyer frontend. Should it be in the frontend toolkit?